jigglypuffRL/common/policies.py

```diff
@@ -13,15 +13,23 @@ class MlpPolicy(BasePolicy):
     """
 
     def __init__(
-        self, state_dim, action_dim, hidden=(32, 32), disc=True,
-        *args, **kwargs
+        self,
+        state_dim,
+        action_dim,
+        hidden=(32, 32),
+        disc=True,
+        det=True,
```

Comment:
Can we remove the deterministic option from the constructor? I'd removed this and added an option to `get_action` because determinsitic action requests can change depending on training and evaluation and does not remain the same throughout.